### ISSUE TITLE
Public key does not exist - bucket URL 404 error

### ISSUE THREAD

**Issue by ricardobrg** [tiki/tiki-sdk-js#109]
Some seconds after initializing the TIKI SDK the browser sends a 404 error for the public.key bucket URL:

https://bucket.storage.l0.mytiki.com/7539e4ae-05da-4192-8151-ddd02a7b8e5a/pKEH8vuIWy6RBpHosfXs0wwTFz3VLAeenaitgWIW_9Y/public.key

This error does not interfere with the usage of the TIKI SDK on the page. All subsequent calls to TIKI SDK work.

### Steps to reproduce
Open this code as index.html in the browser and check the console.
```
<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <title>Document</title>
        <script src="https://unpkg.com/@mytiki/tiki-sdk-js@2.1.2/dist/index.js"></script>
    </head>

    <body>
        <script>
            (async () => {
                console.log('TikiSdk init');
                await TikiSdk.config()
                    .offer
                    .description("offer_desc")
                    .bullet({ text: "1", isUsed: true })
                    .bullet({ text: "2", isUsed: true })
                    .bullet({ text: "3", isUsed: false })
                    .terms("terms")
                    .tag(TikiSdk.Trail.Title.TitleTag.deviceId())
                    .use({ usecases: [TikiSdk.Trail.License.LicenseUsecase.attribution()] })
                    .ptr("test123")
                    .add()
                    .initialize("e12f5b7b-6b48-4503-8b39-28e4995b5f88", "cust23")
                console.log('TikiSdk initialized');
            })()
        </script>
    </body>
</html>
```

### Request
```
Request URL:
https://bucket.storage.l0.mytiki.com/7539e4ae-05da-4192-8151-ddd02a7b8e5a/pKEH8vuIWy6RBpHosfXs0wwTFz3VLAeenaitgWIW_9Y/public.key
Request Method:
GET
Status Code:
404
Remote Address:
[2606:4700:3037::6815:54ed]:443
Referrer Policy:
strict-origin-when-cross-origin
```
### Request headers
```
:Authority:
bucket.storage.l0.mytiki.com
:Method:
GET
:Path:
/7539e4ae-05da-4192-8151-ddd02a7b8e5a/pKEH8vuIWy6RBpHosfXs0wwTFz3VLAeenaitgWIW_9Y/public.key
:Scheme:
https
Accept:
*/*
Accept-Encoding:
gzip, deflate, br
Accept-Language:
pt-BR,pt;q=0.9,en-US;q=0.8,en;q=0.7
Origin:
null
Sec-Ch-Ua:
"Not.A/Brand";v="8", "Chromium";v="114", "Google Chrome";v="114"
Sec-Ch-Ua-Mobile:
?0
Sec-Ch-Ua-Platform:
"macOS"
Sec-Fetch-Dest:
empty
Sec-Fetch-Mode:
cors
Sec-Fetch-Site:
cross-site
User-Agent:
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36
```

### Response Headers
```
Access-Control-Allow-Headers:
*
Access-Control-Allow-Methods:
GET, HEAD, POST, PUT, DELETE, MOVE, OPTIONS
Access-Control-Allow-Origin:
*
Access-Control-Expose-Headers:
Date, Etag, Content-Length, Accept-Ranges, Content-Range, Server, Location, X-Amz-Version-Id
Access-Control-Max-Age:
86400
Alt-Svc:
h3=":443"; ma=86400
Cf-Cache-Status:
DYNAMIC
Cf-Ray:
7ddd4287e9f71d3c-GIG
Content-Encoding:
br
Content-Type:
application/xml
Date:
Tue, 27 Jun 2023 11:13:18 GMT
Nel:
{"success_fraction":0,"report_to":"cf-nel","max_age":604800}
Report-To:
{"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v3?s=fLvSnJ9g%2BhoLmfLiLmdyJYXm2%2BO%2FLLDZmdEm0yYctEFVkwKd4khNdW%2FKoWsnHG4QgonF5xSevY9SwEjD4x8utmLr12H1anhKz5kvZItCdvASdTqSnP7uspOogBU0zVthqDHLuhHYrM%2FUK9Ozb7Z8Ln9eElZe1zM51i3s"}],"group":"cf-nel","max_age":604800}
Server:
cloudflare
X-Amz-Id-2:
chQwyvD51OfcLIdTkHEtRXrHwNkrpwY+pi6eqCXKZUvat2Bzj36IIW5d9CYl2L4Bz6yR4DdYw25P
X-Amz-Request-Id:
1E7F37DEA5CA51D4:B
```

**Comment by mike-audi** [comment]
requires additional investigation.

**Comment by ricardobrg** [comment]
fixed in 2.1.4
